lib/class-wp-theme-json-gutenberg.php

```diff
@@ -669,6 +684,9 @@ private static function compute_preset_vars( $settings ) {
 			$preset_per_origin = _wp_array_get( $settings, $preset['path'], array() );
 			$preset_by_slug    = self::get_merged_preset_by_slug( $preset_per_origin, $preset['value_key'] );
 			foreach ( $preset_by_slug as $slug => $value ) {
+				if ( ! empty( $preset['value_wrapper'] ) ) {
+					$value = str_replace( '$s', $value, $preset['value_wrapper'] );
```

Comment:
I'm not crazy about this approach, but I prefer it to having to define a filter in the theme.json. Any better ideas?

Reply:
It's not too bad in my opinion. Maybe we could use `sprintf` (or `vsprintf`) instead and rename it `value_format` for consistency in terminology. That way if we need to add additional parameters, we could make `value_key` an array and use more than one variable.

Reply:
If we need this, calling it `value_format` and make it work as Alex suggests would be a nice addition.

Reply:
Updated as suggested :)